app/models/traveller.rb

```diff
@@ -0,0 +1,9 @@
+class Traveller < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
+  validates :first_name, presence: true
+  validates :last_name, presence: true
+  validates :origin_planet, presence: true
```

Comment:
@d-tello If we assume, that we are planning only trips from planet `earth`, we should consider killing this column for now. This might cause confusion with some attributes of the Planet model, e.g `distance` assumes that all travellers are on earth.

Reply:
I understood that this was just a fun fact that we had added, such as the planet of origin of the registered traveller, not the place of departure, kind of "nationality".